kernel/include/syscall_handler.h

```diff
@@ -280,8 +281,9 @@ extern int z_user_string_copy(char *dst, const char *src, size_t maxlen);
 #define Z_SYSCALL_VERIFY_MSG(expr, fmt, ...) ({ \
 	bool expr_copy = !(expr); \
 	if (expr_copy) { \
-		printk("syscall %s failed check: " fmt "\n", \
-		       __func__, ##__VA_ARGS__); \
+		LOG_MODULE_DECLARE(os); \
```

Comment:
@nordic-krch is this the right way to route LOG_* in inline functions to the os module?